commit/tokenprice/types.go

```diff
@@ -21,16 +21,32 @@ type Outcome struct {
 type Observation struct {
 	FeedTokenPrices       []cciptypes.TokenPrice                  `json:"feedTokenPrices"`
 	FeeQuoterTokenUpdates map[types.Account]shared.TimestampedBig `json:"feeQuoterTokenUpdates"`
-	FDestChain            int                                     `json:"fDestChain"`
+	FChain                map[cciptypes.ChainSelector]int         `json:"fChain"`
 	Timestamp             time.Time                               `json:"timestamp"`
 }
 
+// AggregateObservation is the aggregation of a list of observations
+type AggregateObservation struct {
+	FeedTokenPrices       map[types.Account][]cciptypes.TokenPrice
+	FeeQuoterTokenUpdates map[types.Account][]shared.TimestampedBig
+	FChain                map[cciptypes.ChainSelector][]int `json:"fChain"`
+	Timestamps            []time.Time
+}
```

Comment:
I know it screws up all of the logic but....
```suggestion
type AggregateObservation []ConsensusObservation
```